nsenter/event.go

```diff
@@ -404,7 +410,29 @@ func (e *NSenterEvent) SendRequest() error {
 	}
 	cmd.Process = process
 
+	//
 	// Transfer the nsenterEvent details to grand-child for processing.
+	//
+
+	// Send the pid using SCM rights, so it shows up properly inside the
+	// nsexec process.
+	//
+	// TODO: in the future we should also send the process uid and gid
```

Comment:
We are already passing uid/gid params in a few syscalls by directly including this info into the nsevent's payload (e.g. overlayfs mounts). Obviously, in those cases we're not entering the backend's user-ns, that's why it works fine. Now, I agree that this approach is more generic/consistent, but you may want to update this comment to reflect that.

Reply:
I updated the comment, thanks!